component/authenticator/src/test/java/org/wso2/carbon/identity/authenticator/smsotp/test/SMSOTPAuthenticatorTest.java

```diff
@@ -664,6 +665,29 @@ public void testGetScreenAttribute() throws UserStoreException, AuthenticationFa
         Assert.assertEquals(smsotpAuthenticator.getScreenAttribute(context,userRealm,"admin"),"******5231");
     }
 
+    @Test
+    public void testGetScreenAttributeWhenMobileRequest() throws UserStoreException {
+        mockStatic(IdentityTenantUtil.class);
```

Comment:
insert new line

Reply:
fixed in https://github.com/wso2-extensions/identity-outbound-auth-sms-otp/pull/144